src/ott/problems/quadratic/quadratic_problem.py

```diff
@@ -138,20 +138,25 @@ def marginal_dependent_cost(
 
     Uses the first term in eq. 6, p. 1 of :cite:`peyre:16`.
 
-    Let :math:`p` [num_a,] be the marginal of the transport matrix for samples
-    from `geom_xx` and :math:`q` [num_b,] be the marginal of the transport
-    matrix for samples from `geom_yy`. `cost_xx` (resp. `cost_yy`) is the
-    cost matrix of `geom_xx` (resp. `geom_yy`). The cost term that
-    depends on these marginals can be written as:
+    Let :math:`p` be the `[n,]` marginal of the transport matrix for samples
+    from `geom_xx` and :math:`q` the `[m,]` marginal of the transport
```

Comment:
Think you can use :attr:\`geom_xx\` (also in other places) and it will link nicely.

Reply:
thanks!!